<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActiveChallenge</class>
 <widget class="QDialog" name="ActiveChallenge">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Испытание</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Настройки</string>
   </property>
   <widget class="QCheckBox" name="cb_final">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Финал</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_timeout">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Авто провал при таймауте</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_start">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Начать испытание</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_success">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Успех</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_fail">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Провал</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_stop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Прервать испытание</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
